src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureActionCell.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, VFC } from 'react';
+import { ActionCell } from 'component/common/Table/cells/ActionCell/ActionCell';
+import { Delete, Undo } from '@mui/icons-material';
+import PermissionIconButton from 'component/common/PermissionIconButton/PermissionIconButton';
+import {
+    DELETE_ADDON,
+    DELETE_FEATURE,
+    UPDATE_FEATURE,
+} from 'component/providers/AccessProvider/permissions';
+import EnvironmentDeleteConfirm from '../../../environments/EnvironmentDeleteConfirm/EnvironmentDeleteConfirm';
+import ArchivedFeatureDeleteConfirm from './ArchivedFeatureDeleteConfirm/ArchivedFeatureDeleteConfirm';
+import { IAddon } from '../../../../interfaces/addons';
+
+interface IReviveArchivedFeatureCell {
+    onRevive: () => void;
+    onDelete: () => void;
+    project: string;
+}
+
+export const ArchivedFeatureActionCell: VFC<IReviveArchivedFeatureCell> = ({
+    onRevive,
+    onDelete,
+    project,
+}) => {
+    return (
+        <ActionCell>
+            <PermissionIconButton
+                onClick={onRevive}
+                projectId={project}
+                permission={UPDATE_FEATURE}
+                tooltipProps={{ title: 'Revive feature toggle' }}
+            >
+                <Undo />
+            </PermissionIconButton>
+            <PermissionIconButton
```

Comment:
Doesn't this need to pass `projectId={project}` as a prop as well? 
Could be nice to make the prop order consistent in both `PermissionIconButton` here.

Reply:
Yes it does. Good catch.